autorun/autorun_compute.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash 
```

Comment:
please move all this out of nevergrad and into dfoptim, since it depends upon dfoptim

Reply:
Done!